client/src/Components/App/index.js

```diff
@@ -1,9 +1,15 @@
-import React from "react";
-import "./index.css";
+import React from 'react';
+import Header from '../CommonComponent/Header/index';
```

Comment:
write only this line without index route :
import Header from '../CommonComponent/Header';